src/Streamly/Internal/FileSystem/File.hs

```diff
@@ -81,6 +81,16 @@ module Streamly.Internal.FileSystem.File
     -- , appendShared
     , appendArray
     , appendChunks
+
+    -- * Deprecated
+    , readWithBufferOf
+    , readChunksWithBufferOf
+    , readChunksFromToWith
+    , toBytes
+    , toChunks
+    , toChunksWithBufferOf
+    , writeWithBufferOf
+    , fromBytesWithBufferOf
```

Comment:
These were not exposed previously?
I don't see any lines highlighted in red?

Reply:
These were renamed earlier without deprecating. Though this is an internal module, this is used by several users so we should deprecate rather than directly removing. You can diff with 0.8.3 to check.